src/keys-builder/template/directive.extractor.ts

```diff
@@ -92,6 +93,8 @@ function resolveKey(ast: OrArray<AST | string>): string[] {
         return resolveKey([expression.trueExp, expression.falseExp]);
       } else if (isLiteralExpression(expression)) {
         return expression.value;
+      } else if (expression instanceof ParenthesizedExpression) {
+        return resolveKey(expression.expression);
```

Comment:
If this is used for unwrapping the ParenthesizedExpression expression, there is a utility function for it
https://github.com/angular/angular/blob/2e3925a65ce4280f8b31c4f8184666880d03acae/packages/core/schematics/utils/typescript/functions.ts#L28
We might be able to use it.